src/main/java/com/sample/Main.java

```diff
@@ -13,7 +13,7 @@
 	public String print() {
 		return "Hello world123.";
 	}
-
+//new branch added
 	public static void main(String args[]) {
 		Main obj=new Main();
```

Comment:
follow proper naming convention 
go through this [link](https://www.geeksforgeeks.org/java-naming-conventions/)